vllm/worker/hpu_model_runner.py

```diff
@@ -373,12 +382,18 @@ def __init__(self, model, vllm_config, is_causal, sampler):
             if self.is_mm_optimized:
                 if hasattr(self.model, 'vision_tower'):
                     self.model.vision_tower = htorch.hpu.wrap_in_hpu_graph(
-                        self.model.vision_tower, disable_tensor_cache=True)
+                        self.model.vision_tower, disable_tensor_cache=False)
```

Comment:
we should not change this as it's not internvl related

Reply:
@SupreetSinghPalne Please check this. it's from https://github.com/HabanaAI/vllm-fork/commit/36eb2cf9cec7ed62d40f29f28f2af48fef31794f